taichi/analysis/verify.cpp

```diff
@@ -54,7 +54,7 @@ class IRVerifier : public BasicStmtVisitor {
           " If it doesn't help, please report this bug by opening an issue at"
           " https://github.com/taichi-dev/taichi to help us improve."
           " Thanks in advance!",
-          stmt->id, op->id);
```

Comment:
ah I feel like stmt->id might still be useful in some cases. Shall we do `{} {} cannot..., stmt->type(), stmt->id()` instead ?

Reply:
Sure. Let me do that.